css/style.css

```diff
@@ -45,7 +45,7 @@ a {
   background: transparent;
   cursor: pointer;
 }
-
+/*comment her*/
 .grid {
   display: flex;
```

Comment:
- [ ] Kindly consider suggesting clear messages and also describing the reason behind your suggestion. so here you can say `I would suggest you to use display: flex here because your class selector named ".grid" and also because the project requirement is to layout your elements with grid here.`